modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/js/blogs.js

```diff
@@ -418,6 +419,7 @@ export default class Blogs {
 						referringPortletResource: this._getElementById(
 							'referringPortletResource'
 						).value,
+						isCreationScreen,
```

Comment:
No need to create a variable, better pass it directly as 

`isCreationScreen: this._getElementById('isCreationScreen').value`

Use single `'` instead of `"`

Anyway, if we need this param only in the back side, couldn't wee get rid of passing it to the js? And just check the logic in java?